src/Inciter/Transporter.C

```diff
@@ -329,7 +332,7 @@ Transporter::load( uint64_t nelem )
 }
 
 void
-Transporter::partition()
+Transporter::part()
```

Comment:
You are correct. However, calling this function is now only a necessary but not sufficient condition for starting partitioning. So keeping the name would be misleading. Do you have a better idea?